client/store/modules/repository/comments/getters.js

```diff
@@ -3,6 +3,7 @@ import get from 'lodash/get';
 import orderBy from 'lodash/orderBy';
 
 export const getComments = state => params => {
+  if (params.contentElementId) params.resolvedAt = null;
```

Comment:
Maybe name should reflect the nature or returning unresolved comments or resolve controlling arg should be passed 🤔